ext.gradle

```diff
@@ -25,7 +25,7 @@
  *  as we want to manage the versions in a single source.
  */
  
-def final SPINE_VERSION = '0.10.75-SNAPSHOT'
+def final SPINE_VERSION = '0.10.76-SNAPSHOT'
```

Comment:
Please make sure that your version is an increment — if you have to merge from `master`.